src/main/java/core/basesyntax/suppliers/BallSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.model.Ball;
+
+public class BallSupplier {
```

Comment:
Looks like this class is not only supplier, it can also print balls. So I think it is better rename class as BallService